network_wrangler/transit/projects/add_route.py

```diff
@@ -284,7 +281,7 @@ def _create_new_frequencies(
         trip_id: Trip ID for the frequencies.
     """
     add_freqs_df = pd.DataFrame(
-        [_parse_headway_record(headway) for headway in headways],
+        [_parse_headway_record(headway)],
```

Comment:
this is just for a single headway record.  Don't you need as many records as there are time ranges?

Reply:
I added a loop [here](https://github.com/wsp-sag/network_wrangler/blob/selection-refactor/network_wrangler/transit/projects/add_route.py#L106) to add trip for each time period. so I changed the headways to headway

Reply:
Ah - I think I remember now that this is function of how we decided to add new routes.  Can you add that to the documentation so it is clear?